src/main/java/core/MyWebDriverManager.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package core;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import io.github.bonigarcia.wdm.config.DriverManagerType;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.lang.reflect.InvocationTargetException;
+
+import static core.utils.LoadEnvironmentFile.getDotenv;
+
+public class MyWebDriverManager {
+    private final DriverManagerType driverManagerType;
+    private static MyWebDriverManager myWebDriverManager;
+    private static WebDriver driver;
+    private static WebDriverWait wait;
+
+    /**
+     * Gets the web driver manager or creates one when needed.
+     *
+     * @param driverManagerType an enum with the driver type
+     * @return an instance of myWebDriverManager
+     */
+    public static MyWebDriverManager getWebDriverManager(final DriverManagerType
+                                                                 driverManagerType)
+            throws ClassNotFoundException, NoSuchMethodException,
+            InvocationTargetException, InstantiationException, IllegalAccessException {
+        if (myWebDriverManager == null) {
+            myWebDriverManager = new MyWebDriverManager(driverManagerType);
+            WebDriverManager.getInstance(driverManagerType).setup();
+            Class<?> driverClass =  Class.forName(driverManagerType.browserClass());
+            driver = (WebDriver) driverClass.getDeclaredConstructor().newInstance();
+            wait = new WebDriverWait(driver, Long.parseLong(getDotenv().get("EXPLICIT_WAIT_TIME")));
+        }
```

Comment:
Maybe, you could also configure the window maximization here.

Reply:
Done.